gammacat/input.py

```diff
@@ -306,6 +347,16 @@ class InputData:
     Expose it as Python objects that can be validated and used.
     """
 
+    def __init__(self, schemas=None, sources=None, datasets=None,
```

Comment:
I shifted the __init__ function to the beginning of the class for better reading. Functionality is not changed (!).